omisego-sdk/src/main/java/co/omisego/omisego/custom/camera/ui/OMGCameraPreview.kt

```diff
@@ -122,7 +123,9 @@ class OMGCameraPreview : SurfaceView, CameraPreviewContract.View {
                 it.setPreviewDisplay(holder)
                 it.setDisplayOrientation(mOMGCameraLogic.getDisplayOrientation(cameraWrapper != null))
                 it.setPreviewCallback(mPreviewCallback)
-                it.startPreview()
+                Handler().postDelayed({
```

Comment:
`SurfaceView extends View`, why not used `postDelayed` directly ?

Reply:
Doesn't even know there's `postDelayed` method in `View` class. Thanks! 👍